frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -107,8 +107,7 @@ function BookInfo:show(file, book_props)
             -- for all others than crengine, we seem to get an accurate nb of pages
             pages = document:getPageCount()
         end
-        -- via pcall because picdocument:getProps() always fails (we could
-        -- check document.is_pic, but this way, we'll catch any other error)
+        -- via pcall, even if getProps() shouldn't fail anymore
```

Comment:
This might be a bit of an unhelpful comment outside of the context of this commit. I'd say just "shouldn't fail" is better than a mysterious "shouldn't fail anymore." Why did it fail before? Should we care?